MapboxCoreNavigation/NavigationRouteOptions.swift

```diff
@@ -34,6 +34,28 @@ open class NavigationRouteOptions: RouteOptions {
         includesExitRoundaboutManeuver = true
     }
 
+    /**
+     Initializes an equivalent `RouteOptions` object from a `NavigationMapOptions`
+     
+     - seealso: `NavigationMatchOptions`
+     */
+    public convenience init(navigationMatchOptions options: NavigationMatchOptions) {
```

Comment:
I see that body of this initializer is like 80% similar to inits on lines 95 and 16. Maybe it's worth moving duplicated lines to some designated initializer or introduce a `commonInit` method?